x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -282,6 +282,11 @@ export const NOTE_URL = '/api/note' as const;
 export const PINNED_EVENT_URL = '/api/pinned_event' as const;
 export const SOURCERER_API_URL = '/api/sourcerer' as const;
 
+export const MATRIX_HISTOGRAM_TEMPLATE_TYPE = `lens`;
+export const MATRIX_HISTOGRAM_TEMPLATE_TAG_TYPE = `tag`;
+export const MATRIX_HISTOGRAM = '/api/matrix_histogram';
```

Comment:
maybe we could name this one _route_ or _path_ (`MATRIX_HISTOGRAM_PATH`)? so it is easier to know what it contains when we use it.